internal/usage/usage.go

```diff
@@ -240,7 +240,7 @@ const (
 	GCPProjectID                    = "Unique identifier of the GCP project in which the network peer resides."
 	Network                         = "Unique identifier of the Network Peering connection in the Atlas project."
 	APIRegion                       = "Indicates which API URL to use, either US or EU. The integration service will use US by default."
-	FormatOut                       = `Output format. 
+	FormatOut                       = `Output format. You can define your own go template by specifying "-o go-template=Template string" or "-o go-template-file=path/to/template" 
 Valid values: json`
```

Comment:
[nit] no need for the long message, also the important piece here which is the valid values is teh only change we need
```suggestion
	FormatOut                       = `Output format. 
Valid values: json|go-template|go-template-file`
```